src/tools/docker-run-to-docker-compose-converter/docker-run-to-docker-compose-converter.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <n-card>
+    <n-form-item label="Your docker run command:">
+      <n-input
+        v-model:value="dockerRun"
+        style="font-family: monospace"
+        type="textarea"
+        placeholder="Your docker run command to convert..."
+        rows="3"
+      />
+    </n-form-item>
+
+    <n-divider />
+
+    <textarea-copyable :value="dockerCompose" language="yaml" />
```

Comment:
Can you update the `textarea-copyable` component to support yaml syntax 

in `src/components/TextareaCopyable.vue`

```
import yamlHljs from 'highlight.js/lib/languages/yaml';
hljs.registerLanguage('yaml', yamlHljs);
```